magda-int-test/src/test/scala/au/csiro/data61/magda/api/DataSetSearchSpec.scala

```diff
@@ -641,7 +669,7 @@ class DataSetSearchSpec extends BaseSearchApiSpec with RegistryConverters {
         }
       }
 
-      it("inexact") {
+      ignore("inexact") {
```

Comment:
I think we should just delete these if we don't need them, can always get them out of git later if we need them.

Reply:
Deleted~ 👍